recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "2.0.2" %}
+{% set version = "2.0.3" %}
 
 package:
   name: libjpeg-turbo
   version: {{ version }}
 
 source:
   url: https://github.com/libjpeg-turbo/libjpeg-turbo/archive/{{ version }}.tar.gz
-  sha256: b45255bd476c19c7c6b198c07c0487e8b8536373b82f2b38346b32b4fa7bb942
+  sha256: a69598bf079463b34d45ca7268462a18b6507fdaa62bb1dfd212f02041499b5d
 
 build:
   number: 1
```

Comment:
Now that you are bumping the version can you reset the build number to `0`?

Reply:
Hmm..I wasn't intending to up the version here..Oh that commit is probably there from a merge from master since it updated since I originally had this PR up. I think build 1 is still apprioriate here.